docs/RELEASE-NOTES.rst

```diff
@@ -20,6 +20,12 @@ Bug Fixes
 * Properly configure http redirect rules on v11.6.1 BIG-IP systems.
 * Failed configurations for objects do not prevent future configurations from happening.
 
+Limitations
+~~~~~~~~~~~
+
+* OpenShift - Does not currently support redirect for individual Routes. If a Route specifies 
+  "insecureEdgeTerminationPolicy" as "Redirect", the http virtual server will enable this policy for all Routes.
```

Comment:
Can we add a link to the open issue at the end of this (much like we do for bug fixes)? Looks good aside from that.

Reply:
Added